src/main/java/uk/gov/pay/directdebit/tasks/services/ExpireService.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.pay.directdebit.tasks.services;
+
+import org.slf4j.Logger;
+import uk.gov.pay.directdebit.app.logger.PayLoggerFactory;
+import uk.gov.pay.directdebit.payments.model.PaymentState;
+import uk.gov.pay.directdebit.payments.model.PaymentStatesGraph;
+import uk.gov.pay.directdebit.payments.model.Transaction;
+import uk.gov.pay.directdebit.payments.services.TransactionService;
+
+import javax.inject.Inject;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Set;
+
+public class ExpireService {
+
+    private TransactionService transactionService;
+    private PaymentStatesGraph paymentStatesGraph;
+    private static final Logger LOGGER = PayLoggerFactory.getLogger(ExpireService.class);
+    private final long MIN_EXPIRY_AGE_MINUTES = 90l;
+    private final PaymentState PAYMENT_EXPIRY_CUTOFF_STATUS = PaymentState.PENDING;
+    
+    @Inject
+    public ExpireService(TransactionService transactionService, PaymentStatesGraph paymentStatesGraph) {
+        this.transactionService = transactionService;
+        this.paymentStatesGraph = paymentStatesGraph;
+    }
+    
+    public int expirePayments() {
+        LOGGER.info("Starting expire payments process.");
+        List<Transaction> paymentsToExpire = getPaymentsForExpiration();
+        int expirePaymentsCounter = 0;
+        for (Transaction payment : paymentsToExpire) {
+            transactionService.paymentExpired(payment);
+            LOGGER.info("Expired payment " + payment.getId());
+            expirePaymentsCounter++;
```

Comment:
is there any reason we use a counter instead of simply returning `paymentsToExpire.size()`?

Reply:
Good point, originally it just returned .size() and so I've reverted back as per your suggestion.